<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>info_movie</class>
 <widget class="QMainWindow" name="info_movie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{border-radius: 10px; background: white; font-size: 15px} QSpinBox, QLineEdit{border-radius: 10px; padding: 5px 10px; font-size: 14px} QSpinBox{} QLabel{font-size: 14px} QCheckBox, QRadioButton{font-size: 15px}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mã phim</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="find_id">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="find_name">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tên phim</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>621</width>
      <height>498</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Mã phim</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tên phim</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Độ tuổi</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Giá vé</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Thời lượng</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Trạng thái</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tạo lúc</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="tNPhimLabel_2">
    <property name="geometry">
     <rect>
      <x>662</x>
      <y>170</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tên phim</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="time">
    <property name="geometry">
     <rect>
      <x>789</x>
      <y>337</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: center</string>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="edit">
    <property name="geometry">
     <rect>
      <x>689</x>
      <y>500</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #4dd4ac</string>
    </property>
    <property name="text">
     <string>Sửa</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>739</x>
      <y>207</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="adult">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>18+</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="child">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Trẻ em</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="middle">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>16+</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="thILNgLabel_2">
    <property name="geometry">
     <rect>
      <x>662</x>
      <y>347</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Thời lượng(phút)</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>629</x>
      <y>27</y>
      <width>31</width>
      <height>621</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="tuILabel_2">
    <property name="geometry">
     <rect>
      <x>669</x>
      <y>227</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Độ tuổi</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="price">
    <property name="geometry">
     <rect>
      <x>789</x>
      <y>267</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="correctionMode">
     <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
   </widget>
   <widget class="QLabel" name="giThPNhTLabel_2">
    <property name="geometry">
     <rect>
      <x>662</x>
      <y>277</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Giá thấp nhất(đ)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hide">
    <property name="geometry">
     <rect>
      <x>839</x>
      <y>500</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #ffda6a</string>
    </property>
    <property name="text">
     <string>Dừng chiếu</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_movie">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>159</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="tuILabel_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>400</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Dừng chiếu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="find_hide">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>110</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #e35d6a</string>
    </property>
    <property name="text">
     <string>Đã dừng</string>
    </property>
   </widget>
   <widget class="QPushButton" name="find_ok">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background:  #4dd4ac</string>
    </property>
    <property name="text">
     <string>Đang phát hành</string>
    </property>
   </widget>
   <widget class="QPushButton" name="find">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #6ea8fe</string>
    </property>
    <property name="text">
     <string>Tìm kiếm</string>
    </property>
   </widget>
   <widget class="QLabel" name="tNPhimLabel_3">
    <property name="geometry">
     <rect>
      <x>662</x>
      <y>111</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mã phim</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idMovie">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>100</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="tuILabel_4">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>450</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Thời gian tạo</string>
    </property>
   </widget>
   <widget class="QLabel" name="createAt">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>450</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="hide_status">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>400</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">margin-left: 2px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
